src/UnitTests/Core/Commands/ScheduleCommandTests/ProcessShould.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using DevChatter.Bot.Core.Commands;
+using DevChatter.Bot.Core.Data;
+using DevChatter.Bot.Core.Events.Args;
+using DevChatter.Bot.Core.Systems.Chat;
+using Moq;
+using System.Collections.Generic;
+using DevChatter.Bot.Core.Data.Model;
+using DevChatter.Bot.Core.Data.Specifications;
+using Xunit;
+
+namespace UnitTests.Core.Commands.ScheduleCommandTests
+{
+    public class ScheduleCommandShould
+    {
+        private readonly ScheduleCommand _scheduleCommand;
+        private readonly Mock<IChatClient> _chatClientMock = new Mock<IChatClient>();
+        private readonly Mock<IRepository> _repositoryMock = new Mock<IRepository>();
+        private readonly CommandReceivedEventArgs _commandReceivedEventArgs = new CommandReceivedEventArgs();
+
+        public ScheduleCommandShould()
```

Comment:
you can use an empty constructor as a way to set up tests. In this case, I put the Mock's `Setup` in here. Since all the tests can work with the same data, I can get away with this. If I wanted to handle different schedules, I'd do that in a separate test class.